Hab2.0_Getting_Started.md

```diff
@@ -0,0 +1,401 @@
+
+```bash
+# Test with existing core package
+sudo hab pkg install core/nginx
+sudo hab svc load core/nginx
+```
+
+### Verification Steps
+
+- Build completes successfully
+- Package artifact is created in `results/` directory
+- Dependencies are pulled from stable channel (verify in build output)
+
+**Note:** Make a note of the atrifact (.hart) file that is generated at the end of the terminal logs.
+
+---
+
+## Step 4: Run the Built Package in Supervisor
+
+### Load and Start the Service
+
+```bash
+
+# Install the package
+sudo -E hab pkg install <path to the artifact(.hart) created in previous step>
+
+# Load the service in supervisor
+sudo hab svc load chef-private/test-service/1.0.0
+
+# Alternatively - Load the service by providing the full pkg_ident
+sudo hab svc load <PKG_IDENT>
+
+# Verify service is loaded
+sudo hab svc status
+```
+
+### Verification Steps
+
+- Service loads without errors
+- Service shows "UP" status in `hab svc status`
+
+---
+
+## Step 5: Setup HAB Auth Token with License Key
+
+### Obtain License Key and Token
+If you don't have one already, generate a free/trial license key to go ahead over [here](https://www.chef.io/license-generation-free-trial). Refer this [page](https://docs.chef.io/habitat/builder_profile/#add-a-progress-chef-license-key) for proper instructions.
+Once generated, you should have the below info ready with you:
+- Valid license key for Habitat 2.0
+- Auth token associated with the license
+
+### Persistent Configuration
+
+Set your HAB_AUTH_TOKEN, if not already set.
+
+```bash
+# Linux/macOS: Add to shell profile
+echo 'export HAB_AUTH_TOKEN=your_auth_token_here' >> ~/.bashrc
+
+# Windows: Set environment variable
+setx HAB_AUTH_TOKEN "your_auth_token_here"
+```
+
+---
+
+## Step 6: Migrate Environment to Habitat 2.0
+
+### (Optional) Stop Current Services
+
+**Note:** If your supervisor is running services while executing the migration script, they will be restarted.
+
+### Run Migration Script
+
+To upgrade a supervisor from 1.6.x to 2.0.x, run the following:
+
+- **linux**: 
+
+```bash
+curl https://raw.githubusercontent.com/habitat-sh/habitat/main/components/hab/migrate.sh | sudo bash -s -- --auth <HAB_AUTH_TOKEN>
+```
+
+- **windows**: 
+
+```bash
+iex "& { $(irm https://raw.githubusercontent.com/habitat-sh/habitat/main/components/hab/migrate.ps1) } --auth <HAB_AUTH_TOKEN>"
+```
+
+### Verification
+- Check version: `hab --version` - Should show hab v2.x
+
+---
+
+## Step 7: Rebuild Plan Against Stable Channel (Habitat 2.0)
+
+### Update Plan File
+Update the `pkg_version="2.0.0"` in the existing plan file.
+
+```bash
+sed -i 's/pkg_version="1.0.0"/pkg_version="2.0.0"/' plan.sh
+```
+
+### Set env to Build Against Stable Channel
+
+```bash
+export HAB_REFRESH_CHANNEL=stable
+export HAB_INTERNAL_BLDR_CHANNEL=acceptance 
+export HAB_STUDIO_SECRET_HAB_INTERNAL_BLDR_CHANNEL=acceptance
+```
+
+### Build
+
+```bash
+hab pkg build .
```

Comment:
use `hab pkg build . --refresh-channel stable` instead of setting `HAB_REFRESH_CHANNEL` above.